solutions/standard/variables.tf

```diff
@@ -192,6 +192,12 @@ variable "existing_kms_instance_crn" {
   description = "The CRN of the KMS instance (Hyper Protect Crypto Services or Key Protect). Required only if `existing_secrets_manager_crn` or `existing_secrets_manager_kms_key_crn` is not specified. If the KMS instance is in different account you must also provide a value for `ibmcloud_kms_api_key`."
 }
 
+variable "is_hpcs_key" {
+  type        = bool
+  default     = false
+  description = "Set it to true if the key provided through the `existing_kms_instance_crn` is Hyper Protect Crypto Services key"
```

Comment:
You know what, we actually don't need this input in the DA. We can use the CRN parser to check if its KP or HPCS by parsing either `existing_kms_instance_crn` (when creating new key) or `existing_secrets_manager_kms_key_crn` when using existing key.

We only need it in the module because its possible user could be creating the KP instance and/or key in same terraform state. But thats not the case in the DA. The `existing_kms_instance_crn` or `existing_secrets_manager_kms_key_crn` will always be known before apply time